src/spamoor_blob/spamoor_blob.star

```diff
@@ -38,6 +46,14 @@ def get_config(
         ),
     ]
 
+    if (
+        "peerdas" in network_params.network
+        or network_params.fulu_fork_epoch != constants.FAR_FUTURE_EPOCH
+    ):
+        image = "ethpandaops/spamoor:blob-v1"
```

Comment:
@barnabasbusa I can't override this image anymore, would it be possible to add `image = spamoor_params.image` to after this to allow override?